definitions.yaml

```diff
@@ -0,0 +1,235 @@
+swagger: "2.0"
+info:
+  description: "This is an [Express](http://expressjs.com/) web application which implements some tasks to automate the provisioning of users in the Digital Citizenship Azure API management _developer portal_.\n\nThe aim is to automate some operations that would otherwise require the manual intervention of the APIs administrator: this lets developers start testing the API just after signing-up.\n\nUser's authentication takes place against an [Azure Active Directory B2C](https://azure.microsoft.com/en-us/services/active-directory-b2c/) tenant."
+  version: "0.1.0"
+  title: Digital citizenship portal
+
+paths: {}
+
+definitions:
+
+  #Returned by GET /subscriptions
+  #Returned by GET /subscriptions/:email
+  SubscriptionCollection: #TODO: add `nextLink` property?
+    type: array
+    items:
+      $ref: "#/definitions/SubscriptionContract"
+
+  #Returned by POST /subscriptions
+  #Returned by POST /subscriptions/:email
+  #Returned by PUT /subscriptions/:subscriptionId/:keyType
+  SubscriptionContract:
+    type: object
+    allOf:
+      - $ref: "#/definitions/Resource"
+      - type: object
+        properties:
+          userId:
+            type: string
+          productId:
+            type: string
+          displayName:
+            type: string
+          state:
+            type: string
+          createdDate:
+            type: string
+            format: "date-time"
+          startDate:
+            type: string
+            format: "date-time"
+          expirationDate:
+            type: string
+            format: "date-time"
+          endDate:
+            type: string
+            format: "date-time"
+          notificationDate:
+            type: string
+            format: "date-time"
+          primaryKey:
+            type: string
+          secondaryKey:
+            type: string
+          stateComment:
+            type: string
+        required:
+          - userId
+          - productId
+          - state
+          - primaryKey
+          - secondaryKey
+
+  Resource:
+    type: object
+    properties:
+      id:
+        type: string
+      name:
+        type: string
+      type:
+        type: string
+
+  #Returned by GET /services/:serviceId
+  Service:
```

Comment:
It seems that when both the definitions of `Service` and `ServicePublic` are added to the file, the `gen-api-models` fails to generate the models:
```
SubscriptionCollection -> src/definitions/onboarding/SubscriptionCollection.ts
SubscriptionContract -> src/definitions/onboarding/SubscriptionContract.ts
Resource -> src/definitions/onboarding/Resource.ts
Service -> src/definitions/onboarding/Service.ts
ServicePublic -> src/definitions/onboarding/ServicePublic.ts
Error: SyntaxError: ',' expected. (7:17)
   5 | /* tslint:disable */
   6 | 
>  7 | import { Service/properties/service_id } from "./Service/properties/service_id";
     |                 ^
   8 | import { Service/properties/service_name } from "./Service/properties/service_name";
   9 | import { Service/properties/organization_name } from "./Service/properties/organization_name";
  10 | import { Service/properties/department_name } from "./Service/properties/department_name";
```
How to deal with such issue?

Reply:
I'll take a look

Reply:
@gunzip any idea?

Reply:
can you try upgrading to italia-utils v4.0.0?

Reply:
I upgraded the library, but the same error still remains.

Reply:
Then it looks like you won this bug! [here's the code of gen-api-model](https://github.com/teamdigitale/io-utils/tree/master/src) - give it a try!